sql/hive/src/test/resources/golden/constant object inspector for generic udf-0-cc120a2331158f570a073599985d3f55

```diff
@@ -1 +1 @@
-{"aa":"10","aaaaaa":"11","aaaaaa":"12","bb12":"13","s14s14":"14"}
+{"aa":"10","aaaaaa":"11","aaaaaa":"12","Bb12":"13","s14s14":"14"}
```

Comment:
We shouldn't change machine generated golden answers though.  If we are going to differ from hive use `checkAnswer` instead.